src/components/Turn/Turn.jsx

```diff
@@ -55,20 +56,21 @@ class Turn extends React.Component {
             currentUsername = this.state.gameState.currentPlayer.username;
         }
         return (
-            <div>
-                <div>Round: {this.state.gameState.roundNum}</div>
-                Players:
-                <ul>
-                    {this.state.gameState.players.map((player, key) => {
-                        return <li key={key}>{player.username}</li>
-                    })}
-                </ul>
-                <div>
-                    {`It's ${currentUsername}'s turn.`}
+            <div className={style.roundInfo}>
+                <h2>Round: {this.state.gameState.roundNum}</h2>
+                <div className={style.currentPlayersBox}>
+                    Players:
+                    <ul>
+                        {this.state.gameState.players.map((player, key) => {
+                            return <li key={key}>{player.username}</li>
+                        })}
+                    </ul>
                 </div>
+                <h2>{`It's ${currentUsername}'s turn.`}</h2>
```

Comment:
Doesn't need to be a string.
```suggestion
                <h2>It's {currentUsername}'s turn.</h2>
```